backend/census_historical_migration/workbooklib/notes_to_sefa.py

```diff
@@ -189,6 +195,60 @@ def xform_missing_notes_records(audit_header, policies_content, rate_content):
     return policies_content, rate_content
 
 
+def xform_missing_notes_records_track_census_data(
```

Comment:
```suggestion
def xform_missing_policies_and_rate_content(
```
I think something like this would align better with how we've been naming xform functions